File: Java/UberVan.java
package Java;

import java.util.ArrayList;
import java.util.Map;

public class UberVan extends Car {
  Map<String, Map<String,Integer>> typeCarAccepted;
  ArrayList<String> seatsMaterial;

  public UberVan(
    String licence, Driver driver,
    Map<String, Map<String,Integer>> typeCarAccepted,
    ArrayList<String> seatsMaterial
  ){
    super( licence, driver);
    this.typeCarAccepted = typeCarAccepted;
    this.seatsMaterial = seatsMaterial;
  }
}

File: PHP/UberVan.php
<?php
require_once('Car.php');

class UberVan extends Car{
  public $typeCarAccepted;
  public $seatsMaterial;

  public function __construct(
    string $licence,
    Driver $driver,
    array $typeCarAccepted,
    array $seatsMaterial
  ){
    parent::__construct( $licence, $driver);
    $this->typeCarAccepted = $typeCarAccepted;
    $this->seatsMaterial = $seatsMaterial;
  }

  public function setPassenger(int $passenger)
  {
    if ($passenger == 6){
      $this->passenger = $passenger;
    }else{
      echo "Necesitas asignar 6 pasajeros";
    }
  }
}